frontend/src/scenes/instance/SystemStatus/BackupTab.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { Button, Card, Input, Select, Tooltip } from 'antd'
+import { useActions, useValues } from 'kea'
+import { systemStatusLogic } from 'scenes/instance/SystemStatus/systemStatusLogic'
+import Paragraph from 'antd/lib/typography/Paragraph'
+
+export function BackupTab(): JSX.Element {
+    const { systemStatus } = useValues(systemStatusLogic)
+    const { createBackup, restoreFromBackup } = useActions(systemStatusLogic)
+    const [ restoreCandidate, setRestoreCandidate] = useState('')
+    const [ backupSuffix, setBackupSuffix] = useState('')
+    const defaultSuffix = "ui"
+
+    return (
+        <>
+            <Card>
+                {console.log("In backup tab")}
+                {systemStatus?.backup.is_enabled ? (
+                    <>
+                        <Tooltip title={`Only alphanumeric-_ are accepted. Default suffix is \"${defaultSuffix}\"`}
+                        >
+                            <Input 
+                                style={{ width: 400 }}
+                                addonBefore="YYYY-MM-DD-HH-mm-" 
+                                placeholder="Optional custom suffix" 
+                                onChange={(value) => setBackupSuffix(value.target.value.replace(/[^\w\-]/g, ''))}
+                            />
+                        </Tooltip>
+                        &nbsp;&nbsp;
+                        <Button
+                            type="primary"
+                            onClick={() => createBackup(`${new Date().toISOString().replace(/[:T]/g, '-').substr(0,17)}${backupSuffix?backupSuffix:defaultSuffix}`)}
+                        >
+                            Create Backup
+                        </Button>
+                        <br/> 
+                        <h2 style={{ color: 'var(--danger)' }}>
+                            Danger Zone
+                        </h2>
+                        <Paragraph type="danger">
+                            This is owerwrites all data in Clickhouse. Please be certain.
```

Comment:
Typo in `owerwrites`

Copy issue: What does "overwriting" mean from the users pov?

Reply:
> What does "overwriting" mean from the users pov?

If I understand correctly, the question is about what happens if someone would click on the resolve button. I didn't necessarily think this through fully yet. When I said overwrite I meant we replace the existing data with the backup data. Note that `clickhouse-backup restore` only works if the table doesn't exist, so if we wanted to do that it would mean we delete the tables first (if they exist) and then restore from the backup.

There are a couple of use-cases, where one would want to do a restore:
1. data got nuked accidentally (no existing tables) - we just need to restore and that's safe
2. new turn-up somewhere (no existing tables) - we can just use the backup and that's safe. Likely here though the tables might get created if we're trying to access the UI.
3. reverting back to an earlier state (existing tables have bad data) - we'll need to delete the tables/posthog database (this is the scary thing to do) and then restore.

Based on the above the interface and access control could be:
1. Restore button limited to admins that only tries to do the restore and fails first if tables exist (potentially we could specify which table to restore / restore all that don't exist)
2. Checkbox for deleting existing tables (by default off), which only the owner can check. Or a separate button for this.

Reply:
Discussed with @fuziontech. 
(2) is the most likely scenario for us to use restore and in that case the tables will still be created as we spin up web, so they must be deleted before. The plan is to make the restore button just error out if tables existed and provide a link do docs that explain how one can delete the tables (i.e. ask users to do the risky action manually) to be able to use this restore button.